packages/config/scripts/checkVerifiedContracts/addresses.ts

```diff
@@ -82,3 +89,18 @@ export function areAllProjectContractsVerified(
     (address) => addressVerificationMap[address.toString()],
   )
 }
+
+function getAddresses(c: ProjectContract): EthereumAddress[] {
+  if (isSingleAddress(c)) {
+    return [c.address]
+  } else {
+    return c.multipleAddresses
+  }
+}
+
+function isSingleAddress(
+  c: ProjectContract,
+): c is ProjectContractSingleAddress {
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  return (c as ProjectContractSingleAddress).address !== undefined
+}
```

Comment:
This TG is repeated

Reply:
I know, but they are in differnet packages with no clear code path to export (the one used in config is used inside script)